x/posts/internal/types/post_test.go

```diff
@@ -608,6 +676,40 @@ func TestPost_Equals(t *testing.T) {
 				Subspace:       "desmos",
 				OptionalData:   map[string]string{},
 				Creator:        otherOwner,
+				Medias:         medias,
+			},
+			expEquals: false,
+		},
+		{
+			name: "Different medias",
```

Comment:
I think this test and the below one should be put inside another functions as they verify only the `PostMedias.Equals` implementation